src/CounterInterceptor/Counts.cs

```diff
@@ -0,0 +1,43 @@
+namespace TemporalioSamples.CounterInterceptor;
+public class Counts
```

Comment:
You are inconsistent with whether you add spacing between namespace and type declaration